codepipeline-tf/providers.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.7"

```

Comment:
I see in version are different in each provider file. make sure to keep same version in each provider file.